CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 ## Unreleased
 
+IMPROVEMENTS:
+
+* Support auto-encrypt [[GH-375](https://github.com/hashicorp/consul-helm/pull/375)].
```

Comment:
Is this not a feature?

Reply:
Oh! haha yes, it is 🤦‍♀

Reply:
Although, I think auto-encrypt is an improvement to existing feature (TLS) and externalServers config is a new feature. I'm not super sure what is the distinction between a feature and an improvement for something that already exists. For example, we called namespaces an improvement, so I'll follow that pattern.